code/game/objects/items/theft_tools.dm

```diff
@@ -105,6 +105,38 @@
 	<li>???</li>\
 	</ul>"
 
+/obj/item/paper/guides/antag/hdd_extraction
+	info = "<h1>Source Code Theft & You - The Idiot's Guide to Crippling Nanotrasen Research & Development</h1><br>\
+	Rumour has it that Nanotrasen are using their R&D Servers to create something awful! They've codenamed it Project Goon, whatever that means.<br>\
+	This cannot be allowed to stand. Below is all our intel for stealing their source code and crippling their research efforts:<br>\
+	<ul>\
+	<li>Locate the physical R&D Server mainframes. Intel suggests these are stored in specially cooled rooms deep within their Science department.</li>\
+	<li>Nanotrasen is a corporation not known for subtlety in design. You should be able to identify the master server by any distinctive markings.</li>\
+	<li>Use the electrical screwdriver with the proprietary tip (provided) to open the server's front panel.</li>\
```

Comment:
Update this paper to not include talk about anything proprietary or anything now that it's basic tools.